platform-hub-api/app/controllers/kubernetes/changeset_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Kubernetes::ChangesetController < ApiJsonController
+
+  before_action :load_changeset
+
+  # GET /kubernetes/tokens-changeset/:cluster
```

Comment:
Nah, that's fine – it's consistent with the other controllers too, and all within the `kubernetes` namespace, so fine for now. (If/when we add Kubernetes functionality that clashes [e.g. changesets management of, say, pods, we can consider renaming then).